tests/conftest.py

```diff
@@ -288,6 +288,8 @@ def init_managers():
     _TaskManagerFactory._build_manager()._delete_all()
     _JobManagerFactory._build_manager()._delete_all()
     _CycleManagerFactory._build_manager()._delete_all()
+    if os.path.exists("./None"):
+        os.remove("./None")
```

Comment:
This line is causing failing test on SQL repo tests. @toan-quach Can you check again?

Error message:
> PermissionError: [WinError 32] The process cannot access the file because it is being used by another process: './None'